pipelines/shopgate.user.deleteFavorites.v1.json

```diff
@@ -0,0 +1,78 @@
+{
+  "version": "1",
+  "pipeline": {
+    "id": "shopgate.user.deleteFavorites.v1",
+    "input": [
+      {"key": "productId", "id": "1"}
+    ],
+    "steps": [
+      {
+        "type": "pipeline",
+        "id": "shopgate.general.getToken.v1",
```

Comment:
I recommend moving this step into the deleteCustomerFavorites pipe, just because we don't want to waste time getting the token if we are not logged in. Same with getFavs I am assuming.